include/audio/AudioSource.h

```diff
@@ -0,0 +1,74 @@
+#pragma once
+#include <string>
+
+#include <AL/al.h>
```

Comment:
I don't think that a library (implementation detail) should be part of Archimedes API.
Includes to other libs should be done only in cpp files or header files that are not part of an our API.

Reply:
However, AL/al.h defines typedefs used later in the file.
C++ modules could fix this problem, but we all know hoe that would work.

Reply:
These types also shouldn't be used in our API.